ogc/src/main/scala/geotrellis/server/ogc/wcs/package.scala

```diff
@@ -45,4 +49,27 @@ package object wcs {
       ymax = self.xmax
     )
   }
+
+  /**
+   * WCS expects this very specific format for its time strings, which is not quite (TM)
+   * what Java's toString method returns.
+   *
+   * The [ISO 8601:2000] syntax for dates and times may be summarized by the following template
+   * (see Annex D of the OGC Web Map Service [OGC 06-042]):
+   * ccyy-mm-ddThh:mm:ss.sssZ
+   * Where
+   * ― ccyy-mm-dd is the date (a four-digit year, and a two-digit month and day);
+   * ― Tis a separator between the data and time strings;
+   * ― hh:mm:ss.sss is the time (a two-digit hour and minute, and fractional seconds);
+   * ― Z represents the Coordinated Universal Time (UTC or ―zulu‖) time zone.
+   *
+   * This was excerpted from "WCS Implementation Standard 1.1" available at:
+   * https://portal.ogc.org/files/07-067r5
+   *
+   * @param zdt
+   */
+  implicit class ZonedDateTimeWcsOps(val zdt: ZonedDateTime) {
+    val formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.'000Z'")
```

Comment:
I think in Java8 ISO formatter is inbuilt: 

```
scala> ZonedDateTime.now().toInstant.toString
res4: String = 2020-02-27T17:33:50.149Z
```

Reply:
Cool, I'll switch to that then